Cargo.toml

```diff
@@ -31,6 +31,7 @@ debug = 1
 opentelemetry = "0.29"
 opentelemetry-appender-tracing = "0.29"
 opentelemetry-http = "0.29"
+opentelemetry-otlp = "0.29"
```

Comment:
do we have to show OTLP exporter in the examples? I think stdout is good enough. Changing from stdout to otlp would be straightforward, and anyway not connected to the instrumentation itself.

Reply:
no problem, switched the examples to stdout and removed the dependency